tests/stubtest_allowlists/py312.txt

```diff
@@ -1,5 +1,4 @@
 # Uncategorised, from Python 3.12
-argparse.BooleanOptionalAction.__init__
```

Comment:
This is a clear gain, in my opinion. Previously, all stubtest checks for this method were disabled, now it just works.